deploy/vagrant/create

```diff
@@ -7,14 +7,14 @@ usage() {
  exit 1
 }
 
-if [[ "$#" != "2" ]]; then
+if [ "$#" -ne 2 ]; then
  usage
 fi
 
 TOTAL="$1"
 PROVIDER="$2"
 
-if [[ "$TOTAL" < "2" ]]; then
+if [ "$TOTAL" -lt 2 ]; then
```

Comment:
I thought it's OK either way, no?
http://www.tldp.org/LDP/abs/html/comparison-ops.html
http://www.tldp.org/LDP/abs/html/quotingvar.html